sdcm/sct_events/database.py

```diff
@@ -136,8 +133,12 @@ def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_even
                                    regex="journal: Suppressed")
 DatabaseLogEvent.add_subevent_type("stream_exception", severity=Severity.ERROR,
                                    regex="stream_exception")
-
-
+DatabaseLogEvent.add_subevent_type("RPC_CONNECTION", severity=Severity.WARNING,
```

Comment:
won't it be published every time we restart some node multiple times flooding WARNING event log?
Is there an easy way of filtering these when SCT triggers restart of some node/server?

Reply:
there is a way, but I don't think all the places we restart/reboot a node, are at the same place.